communitiesuk/funding-service-design-application-store
# FS-3331 adding json format

### Change description
Updated existing `get_applications_statuses_report` function to accept an optional `format` param to allow returning the data in json so we can consume it from assessment (defaults to csv).

Updated the format to breakdown numbers by fund and round so we can populate the assessment dashboard with just one call to application store (rather than one per round).


- [x] README and other documentation has been updated / added (if needed) (confluence: https://dluhcdigital.atlassian.net/wiki/spaces/FS/pages/5238959/Application+report+scripts)


### How to test
Pull this branch and run locally. Then visit the following URLs:
- http://localhost:3002/applications/reporting/applications_statuses_data?format=json Should give you JSON data in the response
- http://localhost:3002/applications/reporting/applications_statuses_data should respond with a csv file as an attachment, as now.
The data in both responses should be the same, just different formats.
